### ISSUE TITLE
Colovesicular fistula

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7447]
**Preferred term label:**
Colovesicular fistula

**Synonyms**
Vesicocolic fistula

**Definition (free text, please give PubMed ID)**
An aberrant, pathological communication between the colon and the bladder.
PMID:30085532

Comment: The most common cause of colovesicular fistulas is the sequelae of complicated diverticulitis and accounts for over two-thirds of cases. The second most common cause is a malignancy in 10% to 20% of cases and is usually adenocarcinoma of the colon. Crohn’s colitis is the third most common cause (5% to 7% of cases) and usually is a result of long-standing disease.
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
